### PR TITLE
Update PharoLauncher.app to version 2.2

### PR DESCRIPTION
**Pull request by andrioli**

### CODE REVIEW

**Comment by estebanlm** [comment]
seems correct, but let's ask the specialist ;)

**Comment by demarey** [comment]
It looks like you took the last commit of the dev branch.
Pharo Launche 2.2 sha is 41fc4b1665cb08bb912633e1638827f9896b1087

File: Casks/pharo-launcher.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask 'pharo-launcher' do
-  version '2.0'
-  sha256 '1da8ef2752b359b804072ba692990e989be4c66d6a761a50ab0693082ac805c7'
+  version '2.2'
+  sha256 '99edeafeeaee64958ef14287791df2228c1d5c8ad83732452d27b471c35dac52'
```
Comment (demarey) [inline Casks/pharo-launcher.rb:3]: It looks like you took the last commit of the dev branch.
Pharo Launche 2.2 sha is 41fc4b1665cb08bb912633e1638827f9896b1087

Comment (andrioli) [reply Casks/pharo-launcher.rb:3]: @demarey this is not the git commit/tag sha

It is the sha256 from the generated binary bundle available at `https://files.pharo.org/pharo-launcher/2.2/PharoLauncher-2.2-x64.dmg`

Comment (demarey) [reply Casks/pharo-launcher.rb:3]: oh, sorry for the misunderstanding. Then it's fine for me. I checked the sha sum.

**Comment by andrioli** [comment]
@estebanlm @demarey can you merge that ?

**Comment by andrioli** [comment]
worked like a charm

Thanks!

```
==> Upgrading pharo-launcher
==> Downloading https://files.pharo.org/pharo-launcher/2.2/PharoLauncher-2.2-x64.dmg
######################################################################## 100.0%
==> Backing App 'PharoLauncher.app' up to '/usr/local/Caskroom/pharo-launcher/2.0/PharoLauncher.app'
==> Removing App '/Applications/PharoLauncher.app'
==> Moving App 'PharoLauncher.app' to '/Applications/PharoLauncher.app'
==> Purging files for version 2.0 of Cask pharo-launcher
🍺  pharo-launcher was successfully upgraded!
```

**Pull request merged**